vsdkx/addon/zoning/processor.py

```diff
@@ -159,38 +168,12 @@ def post_process(self, addon_object: AddonObject) -> AddonObject:
             zone_count[f'zone_{i}'] = obj_class_dict
 
         zone_count[rest_zone_str] = rest_zone_dict
-        zone_count = ZoneProcessor._count_object_ids(zone_count)
 
         inference.extra["zoning"] = zone_count
         addon_object.inference = inference
 
         return addon_object
 
-    @staticmethod
-    def _count_object_ids(zone_dict: dict) -> dict:
-        """
-        Recursive function iterates over zones dictionary and adds count number
-        for every list object value or calls itself for every dictionary value.
-
-        Args:
-            zone_dict (dict): zones dictionary with object class name to object
-            ids mapping.
-
-        Returns:
-            (dict): updated zones dictionary with object class name to object
-            count mapping values added.
-        """
-        zone_dict_copy = zone_dict.copy()
-
-        for key, value in zone_dict.items():
-            if isinstance(value, list):
-                zone_dict_copy[key + '_count'] = len(value)
-            elif isinstance(value, dict):
-                zone_dict_copy[key + '_ids'] = \
-                    ZoneProcessor._count_object_ids(value)
-
-        return zone_dict_copy
-
```

Comment:
why not have it here separately in static function? since it's backwards compatibility functionality maybe it would be better to have it separately in another function ?

Reply:
because the result of this was a nested dictionary with duplicate entries, where `zone_0` is the same as `zone_0_ids`

```
{'tracked_objects': 0, 
'movement_directions': {0: 'downright', 1: 'downleft', 2: 'upleft', 3: '', 4: 'upleft', 5: 'left', 6: 'upleft', 7: 'downright', 8: 'upleft', 9: 'downright', 10: 'left', 11: '', 12: 'downright', 13: 'upleft', 14: ''},
'zoning': {'zone_0': {'Person': [], 'objects_entered': {'Person': []}, 'objects_exited': {'Person': []}}, 'rest': {'Person': [2, 0, 1, 3, 5, 6, 4, 7, 13, 10, 8, 12, 9]}, 'zone_0_ids': {'Person': [], 'objects_entered': {'Person': []}, 'objects_exited': {'Person': []}, 'Person_count': 0, 'objects_entered_ids': {'Person': [], 'Person_count': 0}, 'objects_exited_ids': {'Person': [], 'Person_count': 0}}, 'rest_ids': {'Person': [2, 0, 1, 3, 5, 6, 4, 7, 13, 10, 8, 12, 9], 'Person_count': 13}}}
```

Reply:
That's my mistake, but you could have just changed L.183 to

```py
        zone_dict_copy = {}
```

and this should work ok without duplicates

Reply:
sure, but why change the dictionary keys from `zone_0` to --> `zone_0_ids` ? How's is this method different than what we are doing right now?